lte/gateway/deploy/agw_install_docker.sh

```diff
@@ -10,6 +10,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+MODE=$1
```

Comment:
Not related to just this PR, but given all the underlying work that has already been done with playbooks, what would it take to add new plays for the tasks that the script does so that it can all be run from a remote node helping us scale to deploying and configuring multiple AGWs at the same time?